actors/evm/src/interpreter/instructions/call.rs

```diff
@@ -239,14 +239,16 @@ pub fn call_generic<RT: Runtime>(
                     } else {
                         let method = if !actor_exists
                             || matches!(target_actor_type, Some(Type::Embryo | Type::Account))
-                        // See https://github.com/filecoin-project/ref-fvm/issues/980 for this
-                        // magic value
-                            || (!value.is_zero() && !gas.is_zero() && gas <= U256::from(21_000))
+                            || (!value.is_zero() && gas.is_zero())
```

Comment:
This case also needs to be InvokeContractRestricted. And I _really_ would just call it `Transfer`, or something like that.

Reply:
done.